spec/liquid_validations_spec.rb

```diff
@@ -1,4 +1,4 @@
-require 'spec_helper'
+require_relative 'spec_helper'
```

Comment:
why did you change this?

Reply:
I changed this because it fails to run the test, it gives me a load error like below

![image](https://github.com/espen/liquid-validations/assets/30173722/7c828ff6-61c7-4de0-bbff-f1128a1426d0)